src/dashboards.js

```diff
@@ -10,20 +10,19 @@ const http = require('http');
 const superagent = require('superagent');
 
 
-const CONFIG_PATH = path.join(__dirname,"config.json");
+const CONFIG_PATH = path.join(__dirname, "config.json");
 
 
-function getConfig() {
-    let config = fs.readFileSync(CONFIG_PATH);
+function getConfig(config_path=CONFIG_PATH) {
+    let config = fs.readFileSync(config_path);
     config = JSON.parse(config.toString());
     return config;
 }
 
-function setConfig(key, value, callback) {
-    console.log('setting Config a', key, value)
-    let config = getConfig();
+function setConfig(key, value, callback, config_path=CONFIG_PATH) {
+    let config = getConfig(config_path);
     config[key] = value;
-    fs.writeFile(CONFIG_PATH, JSON.stringify(config), callback);
+    fs.writeFile(config_path, JSON.stringify(config), callback);
```

Comment:
naming convention for js variable
https://github.com/felixge/node-style-guide#naming-conventions

[Use lowerCamelCase for variables, properties and function names](https://github.com/felixge/node-style-guide#use-lowercamelcase-for-variables-properties-and-function-names)
[Use UpperCamelCase for class names](https://github.com/felixge/node-style-guide#use-uppercamelcase-for-class-names)
[Use UPPERCASE for Constants](https://github.com/felixge/node-style-guide#use-uppercase-for-constants)

Reply:
Updated in 9642596